recipes/r-x13binary/meta.yaml

```
@@ -0,0 +1,77 @@
+{% set version = '1.1.39-2' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-x13binary
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/x13binary_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/x13binary/x13binary_{{ version }}.tar.gz
+  sha256: b78383a956061405791439f6991670d2a53626add783cb1f4a391bbbbdf2c8c5
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  skip: True  # [osx]
```

Does not contain compiled code, so the skip can be removed (noarch packages are build on linux after feedstock conversion):
```suggestion
  noarch: generic
```